src/maximumWeightedMatching/maximum_weighted_matching.cpp

```
@@ -0,0 +1,38 @@
+#include "maximum_weighted_matching.hpp"
+#include <boost/graph/adjacency_list.hpp>
+#include <boost/graph/max_cardinality_matching.hpp>
+#include <vector>
+#include <utility>
+
+std::vector<std::pair<int,int>> maximumWeightedMatching(const std::vector<MWEdge>& edges, bool allow_partial)
```

Cppcheck (2.19.0)

[style] The function 'maximumWeightedMatching' is never used.

(unusedFunction)